ftp.go

```diff
@@ -535,6 +535,28 @@ func (c *ServerConn) List(path string) (entries []*Entry, err error) {
 	}
 	return
 }
+//Stat a MLST FTP command
+func (c *ServerConn) Stat(path string)(entry *Entry, err error){
+	var cmd string
+	var parser parseFunc
+
+	if c.mlstSupported {
+		cmd = "MLST"
+		parser = parseRFC3659ListLine
+	} else {
+		return nil, errors.New("MLST unsupported")
+	}
```

Comment:
Please change that block to:
```golang
	if !c.mlstSupported {
		return nil, errors.New("MLST unsupported")
	}
```
And avoid the `cmd` and `parser` variables.

Reply:
ok